Pull request for WP-17-allow-change-meta-package

fblackburn1 opened the pull request:
allow to define a extra repository
add a wazo_meta_package variable

fredericlepied committed:
add a wazo_meta_package variable

fblackburn1 committed:
allow to define a extra repository

fblackburn1 committed:
improve documentation about extra_repository

fredericlepied commented on roles/wazo-vars/defaults/main.yml:
```diff
@@ -4,6 +4,9 @@ wazo_debian_repo: main
 wazo_distribution_upgrade: wazo-dev-buster
+wazo_meta_package: wazo-platform
+# wazo_extra_repository: deb http://mirror.wazo.community/extra/ extra-buster main
+# wazo_extra_repository_filename: wazo-extra
```
we could make it a default to allow the user to only configure wazo_extra_repository

fblackburn1 replied:
I'm not sure how to do this, because we need to have the wazo mirror AND the extra for our use case, not only one or the other

sduthil replied:
I think Fred means to set a real default to the variable `wazo_extra_repository_filename`, so that someone only has to set only `wazo_extra_repository`, instead of both `wazo_extra_repository` AND `wazo_extra_repository_filename`

fblackburn1 replied:
Aah, so yes I can uncomment the line :)

fredericlepied reviewed (changes requested):
see comment

sduthil commented on zuul.d/run.yaml:
```diff
@@ -0,0 +1,54 @@
+---
```
This file should not be in the PR: it was removed in commit 3ddb3b5c2e370d92340392b78673b68339f64869, and it's not related to the PR.

fblackburn1 replied:
Probably a wrong rebase, thanks
Removed

fblackburn1 committed:
set default to wazo_extra_repository_filename

reason: allow to only set wazo_extra_repository variable to define a new
repository
